project/Dependencies.scala

```diff
@@ -208,6 +208,16 @@ object Dependencies {
       ) ++ JacksonDatabindDependencies
   )
 
+  val GoogleBigQuery = Seq(
+    libraryDependencies ++= Seq(
+        "com.typesafe.akka" %% "akka-http" % AkkaHttpVersion,
+        "com.typesafe.akka" %% "akka-http-spray-json" % AkkaHttpVersion,
+        "com.typesafe.akka" %% "akka-stream-contrib" % "0.10",
```

Comment:
We can't depend on `akka-stream-contrib`. The `DelayFlow` shoulded be copied it into this connector.